css/style.css

```diff
@@ -10,7 +10,6 @@ img {
     max-height: 100%;
 }
 
```

Comment:
Make sure to only leave one space line between each style.

- [ ] Correct all multiple spaces between styles.